tools/grab_micro_posts.py

```
@@ -129,11 +148,16 @@
 
     if not json_feed_url or not hugo_content_dir:
-        raise ValueError("NOTES_JSON_FEED_URL and NOTES_HUGO_CONTENT_DIR must be set in the .env file")
+        logging.error("NOTES_JSON_FEED_URL and NOTES_HUGO_CONTENT_DIR must be set in the .env file")
+        return
 
     os.makedirs(hugo_content_dir, exist_ok=True)
 
-    feed_data = download_json_feed(json_feed_url)
+    try:
+        feed_data = download_json_feed(json_feed_url)
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)